bigquery/tests/unit/test_magics.py

```diff
@@ -257,6 +281,48 @@ def test_bigquery_magic_clears_display_in_verbose_mode():
         assert clear_mock.call_count == 1
 
 
+@pytest.mark.usefixtures("ipython_interactive")
+@pytest.mark.skipIf(
+    bigquery_storage_v1beta1 is None, "Requires `google-cloud-bigquery-storage`"
+)
+def test_bigquery_magic_with_bqstorage(monkeypatch):
+    ip = IPython.get_ipython()
+    ip.extension_manager.load_extension("google.cloud.bigquery")
+    mock_credentials = mock.create_autospec(
+        google.auth.credentials.Credentials, instance=True
+    )
+    monkeypatch.setattr(magics.context, "credentials", mock_credentials)
```

Comment:
Why is monkeypatch necessary here?

Reply:
Added a comment. I'm monkeypatching the context so that it's reset after the test completes. It doesn't really matter for `context.credentials`, but it does make a difference for `context.use_bqstorage_api`.